### PR TITLE
Fixes

### LINKED ISSUE #80: Dotted dividers between Life & Arts and Markets, and Markets and Technology

### LINKED ISSUE #81: Make sure editors picks don't contain stories displayed elsewhere.
Engels handles this correctly.

### PR DESCRIPTION
**Pull request by imranolas**
- [x] Dotted borders between topics (fix #80)
- [x] Set line-heights to 1.2
- [x] Remove lead article from editors picks. (fix #81)

### CODE REVIEW

File: client/components/story-card/main.scss [hunk 1]
```diff
@@ -70,6 +72,7 @@
 }
 .story-card__summary {
 	@include nTypeDelta(3);
+	line-height: 1.2em;
 	font-size: 18px;
```
Comment (i-like-robots) [inline client/components/story-card/main.scss]: Is there a reason to mix font size and line height units? Why specify a line height unit at all?

Comment (imranolas) [inline client/components/story-card/main.scss]: `nTypeDelta` sets both the font size and line-height in px which in this case needs to be overridden to fulfil the design requirements. As for the mixed units, it feels more manageable to keep line height as a relative measure proportional to font size, that way we can change the font size without  breaking the line height.

Comment (kaelig) [inline client/components/story-card/main.scss]: In that case I'd recommend a unitless line-height so that children keep the same ratio.

Comment (imranolas) [inline client/components/story-card/main.scss]: Good idea :+1:

Comment (kaelig) [inline client/components/story-card/main.scss]: Friendly note: when the two projects merge (homepage + rest of Next), think about adding new sizes directly into next-type if absolutely needed, as every exception to the rule hinders both design consistency and maintainability.

Comment (imranolas) [inline client/components/story-card/main.scss]: I absolutely agree and that is very much our intention, however, at the moment the designs are in flux and not worth bringing into the rest of Next until they settle down a bit.

Comment (kaelig) [inline client/components/story-card/main.scss]: Great to hear. Totally understand and support your approach.

**Comment by charypar** [comment]
#81 also applies to opinions and topics, this is just a partial fix...

**Pull request merged**